core/src/main/scala/akka/kafka/javadsl/Consumer.scala

```diff
@@ -73,7 +73,7 @@ object Consumer {
    * one, so that the stream can be stopped in a controlled way without losing
    * commits.
    */
-  final class DrainingControl[T] private[javadsl] (control: Control, streamCompletion: CompletionStage[T])
+  final class DrainingControl[T] private[javadsl] (control: Control, val streamCompletion: CompletionStage[T])
```

Comment:
Tiny feature slip. I introduced this to get a handle to the stream completion without calling `drainAndShutdown`.